integreat_cms/static/src/js/media-management/component/directory-content.tsx

```diff
@@ -31,41 +32,43 @@ const DirectoryContent = ({
     setDraggedItem,
     dropItem,
     allowDrop,
+    mediaIsRegional,
 }: Props) => {
     // The file index contains the index of the file which is currently opened in the sidebar
     const [fileIndex, setFileIndex] = fileIndexState;
-
     return (
         <div className="grid grid-cols-gallery max-h-full gap-1">
-            {mediaLibraryContent.map((entry: MediaLibraryEntry, index: number) =>
-                entry.type === "directory" ? (
-                    <Link key={entry.id} href={`/${entry.id}/`} media-library-link>
-                        <DirectoryEntry
-                            directory={entry as Directory}
+            {mediaLibraryContent
+                .filter((content) => !(mediaIsRegional && content.isHidden))
+                .map((entry: MediaLibraryEntry, index: number) =>
+                    entry.type === "directory" ? (
```

Comment:
```suggestion
            {mediaLibraryContent  
                .map((entry: MediaLibraryEntry, index: number) =>
                    (!(!globalEdit && entry.isHidden)) && (
                        entry.type === "directory" ? (
```

This is probably a solution for the bug with the side bar, and we can use `globalEdit` instead defining `mediaIsRegional`.